kube_dns/README.md

```diff
@@ -0,0 +1,60 @@
+# Kube-dns Integration
+
+## Overview
+
+Get metrics from kube-dns service in real time to:
+
+* Visualize and monitor dns metrics collected via Kubernetes' kube-dns addon
+  through Prometheus
+
+See https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dns for
+more informations about kube-dns
+
+## Installation
+
+Install the `dd-check-kube_dns` package manually or with your favorite configuration manager
+
+## Configuration
+
+Edit the `kube_dns.yaml` file to point to your server and port, set the masters to monitor
+
+### Using with service discovery
```

Comment:
I think you can hardcode 10055 instead of port_2. It should be the most common port anyway and if a custom one is used, one can always override with annotation.